exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/VarLenBulkPageReader.java

```diff
@@ -111,6 +112,8 @@ final void set(PageDataInfo pageInfoInput, boolean clear) {
   }
 
   final VarLenColumnBulkEntry getEntry(int valuesToRead) {
+    Preconditions.checkArgument(valuesToRead > 0, "Number values to read cannot be zero");
```

Comment:
Consider `String.format("Number of values to read %d must be positive", valuesToRead);`